### ISSUE TITLE
Reduce package/bundle size - dynamically import borders.json  

### ISSUE THREAD

**Issue by mbrzakovic** [ideditor/country-coder#108]
### Motivation

Currently when taking dependency on country-coder npm package a bundle will get heavier by ~600kb.
The main reason is due to [borders file](https://github.com/ideditor/country-coder/blob/main/src/data/borders.json) which takes 595kb and [is directly imported in source code](https://github.com/ideditor/country-coder/blob/0238f7d8bee9936f2999bd80add6c958beb47f95/src/country-coder.ts#L2).
e.g. running [build analysis in iD](https://github.com/openstreetmap/iD/blob/48f925a27bfb9d9b9e76dc83e7db9f41f74f07a4/package.json#L19) would tell that iD bundle is 3MB while country-coder takes 600kb (~15%), however it seems that geojson is not the best candidate for web-standard compressions (e.g. br) which is why this piece takes ~35% of the compressed bundle.

Making such bundle overhead [impacts fcp](https://web.dev/fcp/) (especially with users that have low speed connection) which can be avoided in case when borders.json is not necessary for startup/fcp. 

### Proposal
Create lazy/dynamic load of the borders file inside the country-coder.
Potential implementation is to have init()/ensureLoaded() method to request+load the heavy file and return the promise which then gets resolved. Unfortunately, that would result in shell pattern (unpopulated object which enables default 'empty' functionalities which become functional after init is done), however in this case it would be worth it.

**Comment by bhousel** [comment]
Yeah, looking at the current stats for our dev RapiD build, this data is the largest chunk at around 663k.  It would be nice to lazy load this like we do with the other data files.

<img width="1611" alt="Screen Shot 2022-11-10 at 10 24 46 AM" src="https://user-images.githubusercontent.com/38784/201135553-794f2925-d573-4020-ab99-9f2d7ffa2310.png">
